nodejs/tests/api.key.test.js

```diff
@@ -0,0 +1,24 @@
+require('dotenv').config();
+console.clear();
+
+const axios = require('axios');
+const urlHandler = require('../handlers/url.handler');
+
+
+// Arkhia config
+const apiKey = process.env.ARKHIA_API_KEY;
+const headers = { 'x-api-key': apiKey };
+
+describe('Test to validate API key', () => {
+    test('Should load API Key', () => {
+        expect(apiKey).toBeDefined();
+        expect(apiKey.length).toEqual(32);
+    });
+
+    test('Should validate API Key', async () => {
```

Comment:
Also a negative test, saying when doing a request with an `invalid` api Key, it should say unauthorised

Reply:
test(`When making a request with invalid ApiKey , should return unauthorised`)

test(`When making a request with no ApiKey, should return `no api key available`)

Reply:
Updated